src/interface/histogram.jl

```diff
@@ -127,8 +127,8 @@ function histogram(
 end
 
 function histogram(x; bins = nothing, closed = :left, kw...)
-    if bins != nothing
-        Base.depwarn("The keyword parameter `bins` is deprecated, use `nbins` instead", :histogram)
+    if bins !== nothing
+        @warn "The keyword parameter `bins` is deprecated, use `nbins` instead"
```

Comment:
Hmmm, any reason for this change? I think the meaning of `depwarn` is quite clear here.

Users can switch to show the depwarn with `julia --depwarn=yes` before upgrading their codes.

Reply:
Sorry was not familiar with `depwarn`. This chang was to fix the test here: https://github.com/Evizero/UnicodePlots.jl/blob/9630e2024a00c08b1f438c4907e036161b91a570/test/tst_histogram.jl#L82.

If tests are run with `--depwarn=yes` in the CI I'm gonna restore that, and add a message in the test file.

Reply:
By default `pkg> test` is triggered with `depwarn=yes` mode. https://github.com/JuliaLang/Pkg.jl/pull/1763

Reply:
Thanks, I've added a message in the test files to reduce confusions, since running with `include("test/runtests.jl")` (not `] test UnicodePlots`) does fails the test unless `--depwarn=yes` is specified.